### DISCUSSION TITLE
Welcome to issues Discussions!

### DISCUSSION THREAD

**Discussion by digitalsadhu** [fabric-ds/issues#40] [Announcements]
## 👋 Welcome!
We’re using Discussions as a place to try to document answers to questions that we get asked about different aspects of Fabric

You can ask questions here if you like or in the Slack channel #finn-fabric. If you ask in Slack, we'll copy, paste and editorialise your question and the subsequent discussion into here.

**Comment by benja** [comment]
Love it!
